.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
+language: scala
+jdk: oraclejdk8
+script:
+- git clone -b 2.12.x https://github.com/scala/scala
```

Comment:
for extra speed you could add a `--single-branch` and a `--depth 1` there too :-)